pkgs/by-name/ru/rustdesk-flutter/package.nix

```diff
@@ -200,7 +200,6 @@ flutter316.buildFlutterApplication rec {
     mkdir -p $out/share/polkit-1/actions $out/share/icons/hicolor/{256x256,scalable}/apps
     cp ../res/128x128@2x.png $out/share/icons/hicolor/256x256/apps/rustdesk.png
     cp ../res/scalable.svg $out/share/icons/hicolor/scalable/apps/rustdesk.svg
-    cp ../res/com.rustdesk.RustDesk.policy $out/share/polkit-1/actions/
```

Comment:
Is this no longer needed or packagers have to provide it?

Reply:
Both. [They stopped shipping it as well](https://github.com/rustdesk/rustdesk/pull/9383/files#diff-b70bfb027a8ef3ec4ab07a53a3b8ba83e63334ba3ce5ca6d9d773900ed452e5e)

Reply:
Oh okay.